### ISSUE TITLE
profile pic placeholder

### ISSUE THREAD

**Issue by alexkaessner** [BreakOutEvent/breakout-frontend#211]
If you have no profile pic, there should be a placeholder to fill the empty space.
![screenshot](https://cloud.githubusercontent.com/assets/6615076/24071766/976119f6-0bda-11e7-9c94-2ad1e61f905d.png)

**Comment by florianschmidt1994** [comment]
Fixed in navbar with aa04c6556bb4b85a8d939ee3cbe92c72c80b3d2c
Related to #166

**Comment by thepiwo** [comment]
@florianschmidt1994 can this be closed?

**Comment by florianschmidt1994** [comment]
No, as the placeholder still is missing in some places that are not the menubar (e.g. in the settings section)
